chain/actors/agen/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+// TODO uncomment when https://github.com/filecoin-project/lotus/pull/6301 merges
```

Comment:
Filed https://github.com/filecoin-project/lotus/pull/6301 that allows Visor to import it here. Until it merges this will need to remain commented out, but that doesn't need to block merging this.